polar-core/src/polar.rs

```diff
@@ -180,6 +180,18 @@ impl Polar {
         self.load(src, None)
     }
 
+    /// Clear rules from the knowledge base
+    pub fn clear_rules(&self) -> PolarResult<()> {
```

Comment:
Does this need to return a result? what's the error case that's getting propagated up? Unless I'm missing something, I only see panics (via the `unwrap()` calls) and success cases.

Reply:
I changed this so it doesn't return a result. The library functions still return 1 or 0 for failure, as panics are converted to failures.